src/components/Correction/AgreementType.vue

```diff
@@ -116,7 +117,7 @@ export default class AgreementType extends Vue {
 
   get selectedAgreementDescription () : string {
     if (this.getAgreementType) {
-      return this.incorporationAgreementTypeResource.find(item => item.code === this.getAgreementType)
+      return AgreementTypeResource.find(item => item.code === this.getAgreementType)
```

Comment:
I tested this code change locally and it continues to work normally:

![image](https://user-images.githubusercontent.com/10002889/167949438-fcf7b413-6f51-435f-9a20-c0b78f71edff.png)